packages/babel-plugin-transform-flow-strip-types/test/fixtures/regression/class-prop-types/options.json

```diff
@@ -1,7 +1,7 @@
 {
   "plugins": [
     "transform-flow-strip-types",
-    "transform-classes",
+    "@babel/plugin-proposal-class-properties",
```

Comment:
This should just be `proposal-class-properties`, to use the package from the monorepo and not from `node_modules`